Makefile

```diff
@@ -96,11 +117,31 @@ $(OC) oc: $(TOOLS_DIR)
 		OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
 		curl -sSLo $(OC) https://mirror.openshift.com/pub/openshift-v4/$${ARCH}/clients/oc/latest/$${OS}/oc.tar.gz ;\
 		tar -xf $(TOOLS_DIR)/oc -C $(TOOLS_DIR) ;\
+		rm -f $(TOOLS_DIR)/README.md ;\
+		$(OC) version ;\
+		version=$(OC_VERSION) ;\
+		$(OC) version | grep -q $${version##v} ;\
 	}
 
 # Install all required tools
 .PHONY: tools
-tools: $(CONTROLLER_GEN) $(KUSTOMIZE) $(OPERATOR_SDK) $(OPM) $(PROMQ)
+tools: $(CONTROLLER_GEN) \
+ 		$(KUSTOMIZE) \
+		$(OC) \
+		$(OPERATOR_SDK) \
+		$(OPM) \
+	 	$(PROMQ)
```

Comment:
should we add golangci-lint  as well to the list?

Reply:
If it is used in the CI, then it makes sense to have it here as well

Reply:
no we don't use it in CI instead we make use of an action 

```yaml
- name: golangci-lint
  uses: golangci/golangci-lint-action@v2
```

Reply:
Ok in that case we can leave it out. I also think that the benefits from caching binaries will be most visible in e2e tests since they're the bottleneck.

Reply:
I believe all jobs that need to download the binaries are running faster .. 
e.g. validate  `make generate`, which needs operator-sdk and controller-gen is faster by around a minute

Reply:
That's true, but e2e tests take 15 minutes and are currently the bottleneck. 
I don't think speeding up linters makes difference for us.